ETWAnalyzer_uTest/EventDump/DumpFileTests.cs

```diff
@@ -404,5 +405,116 @@ public void MinMaxTotalCount()
             Assert.True(dump1.MinMaxFilter(empty));
         }
 
+        
+        [Fact]
+        public void MinMaxTimeFilter()
+        {
+            KeyValuePair<string, MinMaxRange<decimal>>[] RangeValues = new KeyValuePair<string, MinMaxRange<decimal>>[]
+                {
+                    new KeyValuePair<string, MinMaxRange<decimal>>("1s-1000s", new MinMaxRange <decimal>(1, 1000)),
+                    new KeyValuePair<string, MinMaxRange<decimal>>("1000ms-2000ms", new MinMaxRange <decimal>(1000_000/ Million, 2000_000/Million)),
```

Comment:
1000_000/Million = 1, 2000_000/Million is 2 so the value should be 1 and 2 right?

Reply:
corected